syft/presenter/json/artifact.go

```diff
@@ -1,33 +1,135 @@
 package json
 
 import (
+	"encoding/json"
+	"fmt"
+
 	"github.com/anchore/syft/syft/pkg"
-	"github.com/anchore/syft/syft/scope"
+	"github.com/anchore/syft/syft/source"
 )
 
 type Artifact struct {
-	Name      string      `json:"name"`
-	Version   string      `json:"version"`
-	Type      string      `json:"type"`
-	FoundBy   []string    `json:"foundBy"`
-	Locations Locations   `json:"locations,omitempty"`
-	Licenses  []string    `json:"licenses"`
-	Metadata  interface{} `json:"metadata,omitempty"`
+	artifactBasicMetadata
```

Comment:
I think it would be good to add a comment on the `Artifact` struct to explain why it exists, to help readers file it away in the right spot in their mental model.